modules/messages/server/controllers/messages.server.controller.js

```diff
@@ -280,6 +281,16 @@ exports.send = function(req, res) {
       });
     },
 
+    // Here we collect some message data into the influxdb
+    function (message, done) {
+      // the module returns a Promise
+      // it is async, but we'll let it run wild (not waiting for the promise to
+      // be fulfilled or rejected)
+      messageToInfluxService(message);
```

Comment:
ah, ok. i'll try to explain it better there.

the message server response doesn't care about statistics, so there may be no need to wait for that part to fulfill (or worse - respond with an error).
what do you think?